{"name": "Prueba de carga de usuario no encontrado", "status": "passed", "description": "Verifica el manejo de usuarios que no existen en la base de datos", "start": 1747770547481, "stop": 1747770547484, "uuid": "8c4d401b-4d51-4369-beee-6c3612560754", "historyId": "b35e2cd7b1ce882b2585a89120b287f3", "testCaseId": "b35e2cd7b1ce882b2585a89120b287f3", "fullName": "tests.test_eco_guardian.TestLogin#test_load_user_no_encontrado", "labels": [{"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_eco_guardian"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "EdwinFigueroa10"}, {"name": "thread", "value": "26600-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_eco_guardian"}]}